src/OSPSuite.Core/Comparison/PathAndAndValueEntityBuildingBlockDiffBuilder.cs

```diff
@@ -0,0 +1,12 @@
+﻿using OSPSuite.Core.Domain.Builder;
+
+namespace OSPSuite.Core.Comparison
+{
+   public abstract class PathAndAndValueEntityBuildingBlockDiffBuilder<TBuildingBlock, TBuilder> : BuildingBlockDiffBuilder<TBuildingBlock, TBuilder> where TBuilder : PathAndValueEntity where TBuildingBlock : class, IBuildingBlock<TBuilder>
+   {
+      protected PathAndAndValueEntityBuildingBlockDiffBuilder(ObjectBaseDiffBuilder objectBaseDiffBuilder, EnumerableComparer enumerableComparer) : base(objectBaseDiffBuilder, enumerableComparer, x => x.Path)
```

Comment:
x => x.Path is what I needed for equality comparison, but I know I'm about to need this a second time for Individual so there might as well be a common base class.